client/dfclient/include/resources.hpp

```diff
@@ -22,6 +22,7 @@ struct Resources {
 	std::unique_ptr<ncine::AudioBufferPlayer> _wilhelmSound;
 
 	std::vector<tweeny::tween<int>> _tweens;
+	std::map<const uint16_t, tweeny::tween<int>> _mobTweens;
```

Comment:
Nits:
- Unless you need your key-value pairs to be ordered, `unordered_map` should be more efficient
- The key type doesn't need to be const